q2_types/per_sample_sequences/tests/data/absolute_manifests_v2/single-MANIFEST

```diff
@@ -0,0 +1,3 @@
+sample-id	AbsoluteFilepath
```

Comment:
Could you add a test (or modify one of these files) that uses a different valid column header for the sample id column (e.g., `id` or `SampleID`)?

Reply:
Sure, but, is that cross over into "metadata" tests? I left it out because we cover those cases in the metadata subpackage already, not sure if it buys us anything here.

Reply:
I was thinking that this would ensure that these remain metadata-compatible since that's one of the design goals for this format.

Reply:
Cool, so if that is the goal, I will suggest that changing the test data might not be the clearest way to indicate that. What if instead I add a detailed docstring to the formats that details the "metadata-ness" of these formats?

Reply:
That sounds like a good plan.

My goal here was basically to ensure that someone couldn't remove the validation as Metadata from the validator for this format. But looking through the `_validate` method again, that would basically be a complete re-write, so it seems unlikely to happen by accident.